python/cdp/solana_client.py

```diff
@@ -19,26 +32,38 @@ class SolanaClient:
     def __init__(self, api_clients: ApiClients):
         self.api_clients = api_clients
 
-    async def create_account(self, name: str | None = None, idempotency_key: str | None = None):
+    async def create_account(
+        self,
+        name: str | None = None,
+        idempotency_key: str | None = None,
+    ) -> SolanaAccountModel:
```

Comment:
Similarly, remap this to `SolanaAccount`

Reply:
Done.